Contrib/mmpa/test_rfrag.py

```diff
@@ -0,0 +1,25 @@
+#
+#  Copyright (C) 2017 greg Landrum
+#
+#   @@ All Rights Reserved  @@
+#
+import unittest, subprocess, os
+from rdkit import RDConfig
+
+
+class TestCase(unittest.TestCase):
+  def test1Github1406(self):
+    with open('data/simple.smi') as inf:
+      p = subprocess.run(('python', 'rfrag.py'),stdin=inf,stdout=subprocess.PIPE)
```

Comment:
If the sub process blocks on i/o there should be no performance impact other than starting another process.

I think it is one of the few viable ways to run external scripts and it looks like subprocess.getstatusoutput is back from the grave in Python 3.1+